packages/react-components/src/components/Dialpad/Dialpad.tsx

```diff
@@ -199,9 +206,9 @@ const DialpadButton = (props: {
       onLongPress: async () => {
         onLongPress(digit, index);
       },
-      touchEventsOnly: isMobile
+      touchEventsOnly: longPressTrigger === 'touch'
```

Comment:
I think the trigger will have to be `'touch' | 'mouseAndTouuch'` unless we want a way to set it to not trigger for touch events. Currently if its 'mouse' it'll still work for touch

Reply:
yeah I agree this is better. was thinking the third option felt redundant. I like this naming will change it to that